### PR TITLE
[esw] Updated odrive IDs

### PR DESCRIPTION
**Pull request by cgiger00**
resolves #608 

The most updated version of the drive code. Contains the latest odrive IDs and a lot of deprecated features have been removed/cleaned up. 

Integration tested during SAR.

### CODE REVIEW

File: jetson/odrive_bridge/src/__main__.py [hunk 1]
```diff
@@ -39,7 +38,6 @@ def main():
     start_time = t.clock()
 
     legal_controller = int(sys.argv[1])
-    legal_axis = sys.argv[2]
```
Comment (raytitan) [inline jetson/odrive_bridge/src/__main__.py:42]: did we not choose specific axes when booting odrive_bridge before?

Comment (cgiger00) [reply jetson/odrive_bridge/src/__main__.py:42]: at one point i think we did -- we don't anymore its smth that's now deprecated

**Comment by raytitan** [comment]
You're failing the python linter!

**Comment by aswarner** [comment]
@cgiger00 @raytitan do we also need this PR in auton_integration for testing tomorrow?

**Comment by cgiger00** [comment]
is passing the linter!

**Review by raytitan — APPROVED** [review]
This was integration tested during SAR! Looks good to me

The reason changes from already merged auton PR's are showing up in here seem [to be a bug with github.com](https://github.com/isaacs/github/issues/750)

**Review by DavidSmith166 — APPROVED** [review]

File: config/percep/config.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (mmartin4972) [inline config/percep/config.json:1]: Why is this file getting merged in? I thought this was already in main

File: config/percep/project.ini [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[build]
```
Comment (mmartin4972) [inline config/percep/project.ini:1]: Same thing here

File: jetson/nav/gate_search/diamondGateSearch.cpp [hunk 4]
```diff
@@ -13,27 +13,27 @@ DiamondGateSearch::~DiamondGateSearch() {}
 void DiamondGateSearch::initializeSearch()
```
Comment (mmartin4972) [inline jetson/nav/gate_search/diamondGateSearch.cpp:13]: Same thing here. This isn't Odrive related right?

File: jetson/nav/gate_search/gateStateMachine.cpp [hunk 5]
```diff
@@ -10,14 +10,14 @@
 GateStateMachine::GateStateMachine( StateMachine* stateMachine, Rover* rover, const rapidjson::Document& roverConfig )
```
Comment (mmartin4972) [inline jetson/nav/gate_search/gateStateMachine.cpp:10]: Same w this file

**Review by mmartin4972 — CHANGES_REQUESTED** [review]
Where is the Odrive code? This looks like mostly Percep and Nav changes, which should be merged in by Percep and Nav

**Comment by raytitan** [comment]
> 
> 
> Where is the Odrive code? This looks like mostly Percep and Nav changes, which should be merged in by Percep and Nav

@mmartin4972 see my comment

**Review by mmartin4972 — DISMISSED** [review]
Sorry. Just saw Raymond's comments. I found the changed files. Looks good

**Comment by mmartin4972** [comment]
Could you capitalize the U in "updated" in the title and make sure that your commit message looks like this pic: https://github.com/umrover/mrover-workspace/blob/main/admin/workflow_wiki/img/pr_open_a_pull_request.png ? If your description is over 60 char long can you do bullet points?

**Comment by cgiger00** [comment]
@mmartin4972 updated the description lmk if you want anything else added to it

**Review by raytitan — APPROVED** [review]
Looks good, thoroughly reviewed and tested

**Review by DavidSmith166 — DISMISSED** [review]

**Pull request closed without merging**